FirebaseSessions/Sources/SessionInitiator.swift

```diff
@@ -22,8 +26,43 @@ import Foundation
 ///      and comes to the foreground.
 ///
 class SessionInitiator {
+  let sessionTimeout: Double = 1800
```

Comment:
Can write this as `= 30 * 60 // 30 minutes` to increase clarity

Reply:
Also, can we replace Double with time measurements? Eg: TimeInterval rather than a Double. Though technically both are same, calling it TimeInterval will enforce the context more clear.

Reply:
Done